### PR TITLE
Remove all DLLIMPORTGENERATOR_ENABLED usage.

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
/cc @jkoritzinsky @elinor-fung

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

File: src/libraries/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj [hunk 1]
```diff
@@ -138,5 +139,6 @@ System.Net.Http.WinHttpHandler</PackageDescription>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Runtime.InteropServices" />
```
Comment (elinor-fung) [inline src/libraries/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj:142]: Why do we need this? How are there DllImports if it didn't have access to `System.Runtime.InteropServices`?

Comment (elinor-fung) [reply src/libraries/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj:142]: Same question for System.IO.Ports.csproj.

File: src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj [hunk 2]
```diff
@@ -9,6 +9,7 @@
     <!-- Use targeting pack references instead of granular ones in the project file. -->
     <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <EnableDllImportGenerator>true</EnableDllImportGenerator>
```
Comment (elinor-fung) [inline src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj:12]: Was this project being filtered out by for enabling in `generators.targets`? Is this something we should update in how the generator is enabled by default instead?

File: src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/Microsoft.Interop.DllImportGenerator.props [hunk 3]
```diff
@@ -21,7 +21,4 @@
       -->
     <CompilerVisibleProperty Include="DllImportGenerator_UseInternalUnsafeType" />
   </ItemGroup>
-  <PropertyGroup Condition="'$(SetDllImportGeneratorConstants)' != 'false'">
```
Comment (elinor-fung) [inline src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/Microsoft.Interop.DllImportGenerator.props:24]: We should be able to remove `SetDllImportGeneratorConstants` from `generators.targets`.

File: src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj (hunk 2 above)
Comment (AaronRobinsonMSFT) [reply src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj:12]: This is a really wonky project. Adding InteropServices or the other two projects cause a lot of issues references. This was the only way I could make it work.

Comment (elinor-fung) [reply src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj:12]: I suspect it has to do with `DisableImplicitAssemblyReferences` two lines above.  I expect that means it already has the everything referenced. So it is probably actually more 'normal' than the other libraries projects :P

This seems to be the only project that has DisableImplicitAssemblyReferences=false and uses things from Common\Interop (a couple others do seem to have DllImports directly in their own source files), so I guess the one-off is fine.

File: src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertEnumCertificatesInStore.cs [hunk 4]
```diff
@@ -8,12 +8,12 @@ internal static partial class Interop
 {
     internal static partial class Crypt32
     {
-#if DLLIMPORTGENERATOR_ENABLED
+#if UNITTEST
```
Comment (elinor-fung) [inline src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertEnumCertificatesInStore.cs:11]: I thought we were going with explicitly setting `EnableDllImportGenerator` in test projects that used these shared files

**Review by elinor-fung — APPROVED** [review]

**Pull request merged**